xml/admin_operating_pools.xml

```diff
@@ -64,825 +64,821 @@
+<replaceable>crush_ruleset_name</replaceable> <replaceable>expected_num_objects</replaceable></screen>
   <para>
-   This section introduces practical information to perform basic tasks with
-   pools. You can find out how to list, create, and delete pools, as well as
-   show pool statistics or manage snapshots of a pool.
+   The <command>ceph osd pool create</command> can fail if you exceed the
+   limit of placement groups per OSD. The limit is set with the option
+   <option>mon_max_pg_per_osd</option>.
   </para>
+  <variablelist>
+   <varlistentry>
+    <term>pool_name</term>
+    <listitem>
+     <para>
+      The name of the pool. It must be unique. This option is required.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>pg_num</term>
+    <listitem>
+     <para>
+      The total number of placement groups for the pool. This option is
+      required. Default value is 8.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>pgp_num</term>
+    <listitem>
+     <para>
+      The total number of placement groups for placement purposes. This should
+      be equal to the total number of placement groups, except for placement
+      group splitting scenarios. This option is required. Default value is 8.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>crush_ruleset_name</term>
+    <listitem>
+     <para>
+      The name of the crush ruleset for this pool. If the specified ruleset
+      does not exist, the creation of replicated pools will fail with -ENOENT.
+      For replicated pools it is the ruleset specified by the <varname>osd
+      pool default crush replicated ruleset</varname> configuration variable.
+      This ruleset must exist. For erasure pools it is 'erasure-code' if the
+      default erasure code profile is used or
+      <replaceable>POOL_NAME</replaceable> otherwise. This ruleset will be
+      created implicitly if it does not exist already.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>erasure_code_profile=profile</term>
+    <listitem>
+     <para>
+      For erasure coded pools only. Use the erasure code profile. It must be
+      an existing profile as defined by <command>osd erasure-code-profile
+      set</command>.
+     </para>
+     <para>
+      When you create a pool, set the number of placement groups to a
+      reasonable value. Consider the total number of placement groups per OSD
+      too. Placement groups are computationally expensive, so performance will
+      degrade when you have many pools with many placement groups (for example
+      50 pools with 100 placement groups each).
+     </para>
+     <para>
+      See <xref linkend="op-pgs"/> for details on calculating an appropriate
+      number of placement groups for your pool.
```

Comment:
Instead of this paragraph that discusses setting the number of placement groups on pools, I suggest to write something like: "In previous versions of SES, it was necessary to set the number of placement groups on each pool explicitly. In SES7, the Balancer [LINK TO SECTION DESCRIBING THE BALANCER] is active by default, so the number of placement groups is set and adjusted automatically.

Reply:
The link to the PG calculator should have a conspicious CAVEAT saying that calculating and setting PG numbers is only needed if the Balancer is turned off for some reason.